controllers/helmreleaseproxy/helmreleaseproxy_controller.go

```diff
@@ -156,7 +156,7 @@ func (r *HelmReleaseProxyReconciler) Reconcile(ctx context.Context, req ctrl.Req
 				restConfig, err := remote.RESTConfig(ctx, "caaph", r.Client, clusterKey)
 				if err != nil {
 					wrappedErr := errors.Wrapf(err, "failed to get kubeconfig for cluster")
-					conditions.MarkFalse(helmReleaseProxy, addonsv1alpha1.ClusterAvailableCondition, addonsv1alpha1.GetKubeconfigFailedReason, clusterv1.ConditionSeverityError, wrappedErr.Error())
+					conditions.MarkFalse(helmReleaseProxy, addonsv1alpha1.ClusterAvailableCondition, addonsv1alpha1.GetKubeconfigFailedReason, clusterv1.ConditionSeverityError, "failed to get kubeconfig for cluster")
```

Comment:
This could lose some information by not getting the string directly from the `wrappedErr`, and introduces the potential for the two strings with the same error message to disagree. Would it be better to use a temp variable like `errMsg := wrappedErr.Error()` to work around the linter, or just do `//nolint:govet` for these?